+# zrj: add missing link against -lm in libITKNrrdIO-4.10.so
+# NOTYPE  GLOBAL DEFAULT  UND sqrt
+# fixes science/InsightToolkit (binutils 2.27 ld.gold is unhappy)
--- Modules/ThirdParty/NrrdIO/src/NrrdIO/CMakeLists.txt.orig	2016-10-04 18:32:22.000000000 +0300
+++ Modules/ThirdParty/NrrdIO/src/NrrdIO/CMakeLists.txt
@@ -64,7 +64,7 @@ ENDIF(QNANHIBIT)
 ADD_DEFINITIONS(-DTEEM_ZLIB=1)
 
 ADD_LIBRARY(ITKNrrdIO ${nrrdio_SRCS} )
-TARGET_LINK_LIBRARIES(ITKNrrdIO ${ITKZLIB_LIBRARIES})
+TARGET_LINK_LIBRARIES(ITKNrrdIO ${ITKZLIB_LIBRARIES} m)
 
 IF(ITK_LIBRARY_PROPERTIES)
   SET_TARGET_PROPERTIES(ITKNrrdIO PROPERTIES ${ITK_LIBRARY_PROPERTIES})
